.github/workflows/e2e-main-tf.yaml

```
@@ -0,0 +1,145 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: PD E2E Testing Farm
+
+on:
+  push:
+    branches: [main]
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)